src/main/kotlin/race/Record.kt

```diff
@@ -0,0 +1,18 @@
+package race
+
+class Record(raceRecords: List<SnapShot>) {
+    val raceRecords = raceRecords
```

Comment:
생성자의 파라미터와 멤버변수 선언을 동시해 해보는 것은 어떨까요?
```suggestion
class Record(val raceRecords: List<SnapShot>) {

```